_docs/payout-details.md

```diff
@@ -238,8 +259,10 @@ For GBP IBAN transfers please use:
 </div>
 
 <div class="alert alert-info" markdown="1">
-**Note!** Transfer is done using the fastest method available on the recipient's bank.
+**Note!**
 
+* You can only make use of one of the above options at a time, i.e you can only use either of GBP Faster payment or GBP IBAN transfer method but not both.
```

Comment:
Both of them will use Faster Payments. You should specify that the customer has either need to enter an IBAN (And an optional BIC), or an account number and sort code